lib/config/base.js

```diff
@@ -15,5 +23,30 @@ module.exports = {
     es6: true,
   },
 
+  plugins: ['decorator-position'],
+}
+
+
+const esLint8Config = {
+  root: true,
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    ecmaFeatures: {
+      legacyDecorators: true,
+    },
+    babelOptions: {
+      configFile: resolve(__dirname, '../../.babelrc'),
```

Comment:
I don't think we can know what file path the .babelrc will be at -- is there a generic way of specifying this?